docs/source/getting-started/tracking-server-overview/index.rst

```diff
@@ -0,0 +1,327 @@
+.. _tracking-server-overview:
+
+5 Minute Tracking Server Overview
+======================================================================
+
+In this guide we will walk you through how to view your MLflow experiment results with different types of 
+tracking server. At a high level there are 3 ways to view your MLflow experiments:
+
+* **[Method 1]** Start your own MLflow server.
+* **[Method 2]** Use a free hosted tracking server - Databricks Community Edition.
+* **[Method 3]** Use production Databricks/AzureML.
+
+To choose among these 3 methods, here is our recommendation:
+
+* If you have privacy concerns (data/model/tech stack), use **Method 1 - start your own server**.
+* If you are a student/individual researcher, or if you are developing in cloud-based notebook, e.g., Google 
+  Colab, use **Method 2 - free hosted tracking server**.
+* Enterprise users, or if you want to serve/productionize your model, please use 
+  **Method 3 - production Databricks/AzureML**.
+
+Overall **Method 2 - free hosted tracking server** is the simplest way to get started with MLflow, but please
+pick the method that best suits your needs.
+
+Method 1: Start Your Own MLflow Server
+---------------------------------------
+
+**Disclaimier**: This part of guide is not suitable for running in a cloud-provided IPython environment, e.g., 
```

Comment:
can we wrap the exempli gratia statements in parentheses and just use a comma separator between the short list of examples? 
(e.g., Collab, Databricks)